fiware-region-sanity-tests/commons/constants.py

```diff
@@ -26,6 +26,7 @@
 
 # CONFIGURATION PROPERTIES
 PROPERTIES_FILE = "resources/settings.json"
+PROPERTIES_LOGGER = "TestCase"
```

Comment:
It is only an opinion, but our unittest class used by tests is *FiwareTestCase*, so that all test cases are "FiwareTestCase". I think it is more representative another name for the logger used in these ones, but **LGTM**